### ISSUE TITLE
webview: TypeError: Cannot set property 'scrollTop' of null

### ISSUE THREAD

**Issue by bkraul** [RocketChat/Rocket.Chat.Electron#1238]
## My Setup
- Operating System: Windows 10 (1809)
- App Version: 2.15.3
- Installation type: exe

- [x] I have tested with the latest version
- [x] I can simulate the issue easily

## Description
Upon upgrading RocketChat server, the last displayed conversation goes blank. All other conversations are fine. The developer tools window displays the following:

I guessed it had something to do with cache, but I did "Reload ignoring cache" and it did not help. This does not happen on the web-based access to the rocketchat endpoint, only on the Electron client.

```
webview: TypeError: Cannot set property 'scrollTop' of null
    at https://myserver.domain.com/9b216696b264cec2c24a4e68ee99e13b19a294ba.js?meteor_js_resource=true:281:282689
    at t._compute (https://myserver.domain.com/9b216696b264cec2c24a4e68ee99e13b19a294ba.js?meteor_js_resource=true:43:2321)
    at new t (https://myserver.domain.com/9b216696b264cec2c24a4e68ee99e13b19a294ba.js?meteor_js_resource=true:43:1452)
    at Object.s.autorun (https://myserver.domain.com/9b216696b264cec2c24a4e68ee99e13b19a294ba.js?meteor_js_resource=true:43:3998)
    at L (https://myserver.domain.com/9b216696b264cec2c24a4e68ee99e13b19a294ba.js?meteor_js_resource=true:281:281867)
    at g.action [as _action] (https://myserver.domain.com/9b216696b264cec2c24a4e68ee99e13b19a294ba.js?meteor_js_resource=true:281:207847)
    at g.callAction (https://myserver.domain.com/9b216696b264cec2c24a4e68ee99e13b19a294ba.js?meteor_js_resource=true:161:10091)
    at https://myserver.domain.com/9b216696b264cec2c24a4e68ee99e13b19a294ba.js?meteor_js_resource=true:161:6845
    at Object.s.nonreactive (https://myserver.domain.com/9b216696b264cec2c24a4e68ee99e13b19a294ba.js?meteor_js_resource=true:43:4179)
    at https://myserver.domain.com/9b216696b264cec2c24a4e68ee99e13b19a294ba.js?meteor_js_resource=true:161:6692
```
![image](https://user-images.githubusercontent.com/14616851/58250408-f9854200-7d26-11e9-92f6-25ccf2d3aab3.png)

### Current Behavior
Screen goes blank on last conversation.

### Expected Behavior
The last conversation should be properly loaded.

**Comment by bkraul** [comment]
This happened while upgrading the server to develop commits [5e60a8a](https://github.com/RocketChat/Rocket.Chat/commit/5e60a8a113365f228899ca82cfcaca0bec680a03) and [482ae2d](https://github.com/RocketChat/Rocket.Chat/commit/482ae2d644a3886af2b65bb2b04b4ec63bd15892) (latest develop).

Client does not seem to have this issue with commit [309335e](https://github.com/RocketChat/Rocket.Chat/commit/309335e865f629b3bc39f2f7c395f3af1fd74803)

**Comment by bkraul** [comment]
Seems there are more errors:
```
webview: Exception from Tracker recompute function:
webview.js:52 webview: Error: Expected template rendered with Blaze.render
    at Object.f.remove (https://kaiwa.epginsurance.com/9b216696b264cec2c24a4e68ee99e13b19a294ba.js?meteor_js_resource=true:87:24512)
    at f.close (https://kaiwa.epginsurance.com/9b216696b264cec2c24a4e68ee99e13b19a294ba.js?meteor_js_resource=true:281:260133)
    at Array.from.map.t (https://kaiwa.epginsurance.com/9b216696b264cec2c24a4e68ee99e13b19a294ba.js?meteor_js_resource=true:281:260398)
    at Array.map (<anonymous>)
    at f.closeOlderRooms (https://kaiwa.epginsurance.com/9b216696b264cec2c24a4e68ee99e13b19a294ba.js?meteor_js_resource=true:281:260386)
    at f.open (https://kaiwa.epginsurance.com/9b216696b264cec2c24a4e68ee99e13b19a294ba.js?meteor_js_resource=true:281:260630)
    at https://kaiwa.epginsurance.com/9b216696b264cec2c24a4e68ee99e13b19a294ba.js?meteor_js_resource=true:281:282018
    at t._compute (https://kaiwa.epginsurance.com/9b216696b264cec2c24a4e68ee99e13b19a294ba.js?meteor_js_resource=true:43:2321)
    at new t (https://kaiwa.epginsurance.com/9b216696b264cec2c24a4e68ee99e13b19a294ba.js?meteor_js_resource=true:43:1452)
    at Object.s.autorun (https://kaiwa.epginsurance.com/9b216696b264cec2c24a4e68ee99e13b19a294ba.js?meteor_js_resource=true:43:3998)
```

**Comment by bkraul** [comment]
@tassoevan is there any kind of process/migration script I would have to run to correct this? I just find it odd that the web client has no issues.

**Comment by tassoevan** [comment]
@bkraul Could you please check which values are listed on `emoji.recent` variable (`Application > Local storage`)? I've seen a similar issue after EmojiOne change to JoyPixels.

**Comment by bkraul** [comment]
Here it is:
```
ohmy,triumph,scream,fingers_crossed,thumbsup,nauseated_face,rubber_duck,gumo,ocasio,pelosi,aoc,sekiro_death,poop,pbear,amiwrong,dizzy,trump_eyeroll,middle_finger,ayaya,rolling_on_the_floor_laughing,filter,jobs,trump,+1_tone3,arrow_upper_right,cry,sunglasses,musical_note,crying_cat_face,smirk,maga,klansman,takei,laughing,pedobear,kkk,scream_cat,mueller,smile,cat,frowning2,rainbow_flag,wink,rolling_eyes,dog2,smile_cat
```
I have a few custom emoji in this list. Would I need to be disabling some of those?

**Comment by bkraul** [comment]
@tassoevan , would there be any chance that the original issue is also server-related, where the object ends up being null?

**Comment by bkraul** [comment]
I have updated to develop build based on commit [53093e0](https://github.com/RocketChat/Rocket.Chat/commit/53093e0c6e1d9d902bb44caeb1bc23a193cd8596). Everything seems to be working with no further blank pages. Just for kicks I tried to go back to [482ae2d](https://github.com/RocketChat/Rocket.Chat/commit/482ae2d644a3886af2b65bb2b04b4ec63bd15892), but apparently there is a migration version change in between, and could no longer go back. (which means phew...thank God it works now, as I have a new base).
